docs/intro.md

```diff
@@ -1,22 +1,89 @@
+Jupyter Book 2.0 allows users to
+
+* write their content in [markdown files](https://myst-parser.readthedocs.io/en/latest/) or [Jupyter](https://jupyter.org/) notebooks,
+* include computational elements (e.g., code cells) in either type,
+* include rich syntax such as citations, cross-references, and numbered equations, and
+* using a simple command, run the embedded code cells, [cache](https://jupyter-cache.readthedocs.io/en/latest/) the outputs and convert this content into:
+    * a web-based interactive book and
+    * a publication-quality PDF.
 
 ```{warning}
-This is an early prototype tool that may evolve quickly. Your feedback is
-very welcome! To give it, please [open an issue](https://github.com/ExecutableBookProject/cli/issues/new)
+This is an early prototype that may evolve quickly. All [feedback](https://jupyter-cache.readthedocs.io/en/latest/) is very welcome! 
 ```
 
+
+## A Small Example Project
+
+[Here's](https://executablebookproject.github.io/quantecon-mini-example/docs/index.html)
+short example of a web-based book created by Jupyter Book 2.0.
+
+Some of the features on display include
+
+* [Jupyter notebook-style inputs and outputs](https://executablebookproject.github.io/quantecon-mini-example/docs/python_by_example.html#version-1)
+* [citations](https://executablebookproject.github.io/quantecon-mini-example/docs/about_py.html#bibliography)
+* [numbered equations](https://executablebookproject.github.io/quantecon-mini-example/docs/python_by_example.html#another-application)
+* [numbered figures](https://executablebookproject.github.io/quantecon-mini-example/docs/getting_started.html#jupyter-notebooks) with captions and cross-referencing
+
+The source files are written in [Myst markdown](https://myst-parser.readthedocs.io/en/latest/) and can be [found
+on GitHub](https://github.com/ExecutableBookProject/quantecon-mini-example/)
+in the [docs](https://github.com/ExecutableBookProject/quantecon-mini-example/tree/master/mini_book/docs) directory.
+
+(The could alternatively have been written directly as Jupyter notebooks.)
+
+The MyST source files can be edited as either text files or Jupyter notebooks,
+or a combination of both, via [Jupytext](https://jupytext.readthedocs.io/en/latest/introduction.html)
+
+Note that 
+
+* Jupytext [supports](https://jupytext.readthedocs.io/en/latest/formats.html#myst-markdown) the Myst markdown format.
+
+* If editing the markdown files using VS Code, the [MyST markdown extension](https://marketplace.visualstudio.com/items?itemName=ExecutableBookProject.myst-highlight) provides syntax highlighting and other features.
+
+
+You can build this book locally on the command line via the following steps:
+
+1. Ensure you have a recent version of [Anaconda Python](https://www.anaconda.com/distribution/) installed.
+
+2. Install the Python libraries needed to run the code in this particular example:
+
+    ```
+    conda install numpy matplotlib
+    ```
+
+3. Install Jupyter Book 
```

Comment:
I don't think this is the proper way to install in the root conda environment using pip. Generally I have to find the pip package in the conda root environment and pip install in that path. Are you able to build the project from the root environment following these steps?